combinations/hash.tsv

```diff
@@ -93,7 +93,7 @@ python=3.7
 python=3.8
 python=3.9
 samtools=1.11,bowtie=1.3.0
-r-base=4.0,r-optparse=1.6.6,r-ggplot2=3.3.2,bioconductor-txdb.hsapiens.ucsc.hg19.knowngene=3.2.2=r40_9,bioconductor-edger=3.32.0,bioconductor-grohmm=1.24.0,bioconductor-org.hs.eg.db=3.12.0
+r-base=4.0.3,r-optparse=1.6.6,r-rmariadb=1.1.0,bioconductor-edger=3.32.0,bioconductor-grohmm=1.24.0,bioconductor-org.hs.eg.db=3.12.0,bioconductor-txdb.hsapiens.ucsc.hg19.knowngene=3.2.2
```

Comment:
can you please add a new line with your packages?

Reply:
Yes of course, I made the container a new line!